src/plugins/visualizations/public/types.ts

```diff
@@ -71,4 +73,7 @@ export interface VisToExpressionAstParams {
   abortSignal?: AbortSignal;
 }
 
-export type VisToExpressionAst = (vis: Vis, params: VisToExpressionAstParams) => string;
+export type VisToExpressionAst = (
```

Comment:
:information_source: This was completely wrongly typed :-) This function was never supposed to return a `string`. It's used in [`build_pipeline`](https://github.com/elastic/kibana/blob/master/src/plugins/visualizations/public/legacy/build_pipeline.ts#L446) and the result is passed in a function expecting an ExpressionAst (also all implementations currently return that).